File: lab1-semester1/3.cpp
/*3.       Write a program to demonstrate the use of cin and cout statements.
(Hint: take input from user.)*/

#include<iostream>
using namespace std;

int main(){
    string fullName;
    string id;
    cout<<"Input your full name : ";
    getline (cin, fullName);//this gets both name and surname in a time
    
    cout<<"Input your id : ";
    cin>>id;
    
    cout<<"\n Your name is : "<<fullName<<endl;
    cout<<"\n Your id is : "<<id<<endl;
    
    system("pause");
    return 0;
}

File: lab1-semester1/3.go
/*3.       Write a program that can take input from user and print it*/

package main
  
import "fmt"
  
func main() {
  
    fmt.Println("Enter Your First Name: ")

    // var then variable name then variable type
    var first string
    // Taking input from user
    fmt.Scanln(&first)

    fmt.Println("Enter Your Last Name: ")
    var second string
    fmt.Scanln(&second)

fmt.Println("Enter Your ID: ")
    var id string
    fmt.Scanln(&id)
  
    // Print function is used to
    // display output in the same line
    fmt.Print("Your Full Name is: ", first + " " + second+" "+id)
 
}
